src/schemas/raster.schema.json

```diff
@@ -43,22 +43,58 @@
     "metadata": {
       "type": "object",
       "additionalProperties": false,
-      "required": ["dimensions", "is_pyramid", "transform"],
+      "required": [],
       "properties": {
         "dimensions": { "$ref": "#/definitions/dimensions" },
         "is_pyramid": { "type": "boolean" },
-        "transform": { "$ref": "#/definitions/transform" }
+        "transform": { "$ref": "#/definitions/transform" },
+        "omeTiffOffsetsUrl": { "type": "string", "format": "uri" }
+      }
+    },
+    "requestInit": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [],
+      "properties": {
+        "method": {
+          "type": "string"
+        },
+        "headers": {
+          "type": "object"
+        },
+        "body": {
+          "type": "string"
+        },
+        "mode": {
+          "type": "string"
+        },
+        "credentials": {
+          "type": "string"
+        },
+        "cache": {
+          "type": "string"
+        },
+        "redirect": {
+          "type": "string"
+        },
+        "referrer": {
+          "type": "string"
+        },
+        "integrity": {
+          "type": "string"
+        }
       }
```

Comment:
This should really be an optional field in metadata. That key is meant to hold any image-type -specific metadata. (Which the tiff offsets are).

Reply:
The offsets are in the metadata.  These are the request headers - it didn't seem these should go under metadata since other schemas could use them for fetching further data and they won't necessarily have a `metadata` section.